### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by dreispt** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1:

File: __unported__/project_service_type/__openerp__.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 This will add a type of service on project allowing you to categorize them.
 """,
      "website": "http://camptocamp.com",
+     "license": "GPL-3 or any later version",
```
Comment (StefanRijnhart) [inline __unported__/project_service_type/__openerp__.py]: Header of this file actually says GPL-2 or any later version.

Comment (dreispt) [inline __unported__/project_service_type/__openerp__.py]: So we need to ask @jgrandguillaume if he agrees to relicense it as AGPL-3.

**Comment by max3903** [comment]
:+1:

File: __unported__/project_service_type/__openerp__.py (hunk 1 above)
Comment (yvaucher) [inline __unported__/project_service_type/__openerp__.py]: I changed it to comply with header. This can be re-licensed when porting it.

**Pull request merged**